go/libkb/checkers.go

```diff
@@ -11,7 +11,9 @@ import (
 )
 
 var emailRE = regexp.MustCompile(`^\S+@\S+\.\S+$`)
-var usernameRE = regexp.MustCompile(`^([a-zA-Z0-9][a-zA-Z0-9_]?)+$`)
+
+// Disallow multiple underscores in a row.
+var usernameRE = regexp.MustCompile(`^([a-zA-Z0-9]+[a-zA-Z0-9_]?)+$`)
```

Comment:
Adding the inner + makes the regexp clearer IMO, and translates directly to the (greedy) grammar syntax